src/sagemaker/fw_utils.py

```diff
@@ -23,14 +23,18 @@
 
 """This module contains utility functions shared across ``Framework`` components."""
 
-
 UploadedCode = namedtuple('UserCode', ['s3_prefix', 'script_name'])
 """sagemaker.fw_utils.UserCode: An object containing the S3 prefix and script name.
 
 This is for the source code used for the entry point with an ``Estimator``. It can be
 instantiated with positional or keyword arguments.
 """
 
+EMPTY_FRAMEWORK_VERSION_WARNING = 'In an upcoming version of the SageMaker Python SDK, ' \
+                                  'framework_version will be required to create an estimator. ' \
+                                  'Please add framework_version={} to your constructor to avoid ' \
```

Comment:
to  the constructor instead of to your constructor?

Doesn't really matter though.